### PR TITLE
[script.xbmcbackup] 1.1.3

### PR DESCRIPTION
**Pull request by robweber**
### Description

Wanted to push the most current master branch of this repo before starting to port to Python 3 for future versions of Kodi. Notable improvements since 1.1.2 are: 

* added file chunk support for dropbox uploads
* fixed settings duplicate ids
* added scheduler delay to assist with time sync (rpi mostly)
* fixed some minor coding issues to better align with Kodi repo standards and Kodi Python API

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.xbmcbackup/addon.xml [hunk 1]
```diff
@@ -16,76 +16,85 @@
...
+    <description lang="sk_SK">Už ste niekedy poškodili konfiguráciu Kodi a priali si mať zálohu? Teraz môžete - na jeden klik. Môžete exportovať Vašu databázu, playlist, náhľady, doplnky a konfigurácie na ktorýkoľvek zdroj zapisovateľný Kodi. Zálohy môžu byť púšťané na požiadanie alebo plánovačom. </description>
+    <description lang="sv_SE">Har du någonsin tappat bort din Kodi konfiguration och önskat att du hade en backup? Nu kan du enkelt med ett klick. Du kan exportera din databas, spellista, minityrer, tillägg och andra konfigurationsdetaljer till valfri källa som är skrivbar för Kodi. Backupper kan köras på begäran eller via scheman.</description>
+    <description lang="zh_CN">你是否经常折腾你的 Kodi，因而希望能够有个备份？你可以把资料库、播放列表、缩略图、插件和其他配置细节导出到 Kodi 可以写入的任意位置。备份可以按需运行或通过计划任务执行。</description>
     <language></language>
     <platform>all</platform>
     <license>The MIT License</license>
-    <forum>http://forum.xbmc.org/showthread.php?tid=129499</forum>
+    <forum>https://forum.kodi.tv/showthread.php?tid=129499</forum>
     <source>https://github.com/robweber/xbmcbackup</source>
+    <email></email>
```
Comment (Rechi) [inline script.xbmcbackup/addon.xml]: drop the empty tag

File: script.xbmcbackup/addon.xml [hunk 2]
```diff
@@ -16,76 +16,85 @@
...
+    <assets>
+    		<icon>resources/media/icon.png</icon>
```
Comment (Rechi) [inline script.xbmcbackup/addon.xml]: This isn't supported in jarvis, it uses `<ROOT-DIR>/icon.png` only.

File: script.xbmcbackup/addon.xml (hunk 1 above)
Comment (robweber) [reply script.xbmcbackup/addon.xml]: Done - for reference I was following [the wiki](https://kodi.wiki/view/Addon.xml#.3Cemail.3E) and it sounded like it should be there but left empty.

**Comment by Rechi** [comment]
@robweber this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Pull request merged**